crates/sallyport/src/guest/call/enarxcall/park.rs

```diff
@@ -7,25 +7,23 @@ use crate::item::enarxcall::Number;
 use crate::libc::timespec;
 use crate::{Result, NULL};
 
-#[derive(Debug, Clone, Copy)]
-pub struct ParkTimeout {
-    pub timespec: timespec,
-    pub absolute: bool,
-}
+use core::ffi::c_int;
 
+#[derive(Debug, Clone, Copy)]
 pub struct Park<'a> {
-    pub timeout: Option<&'a ParkTimeout>,
+    pub expected_val: c_int,
```

Comment:
nit; `expected` would probably suffice and be consistent with the shim implementations. This is good enough though